### ISSUE TITLE
/sga:admin add constallation can soft brick world

### ISSUE THREAD

**Issue by BeeFox-sys** [Nettakrim/Spyglass-Astronomy#7]
Steps to replicated:
1. Create new world
2. run command `/sga:admin add constellation 1 2 3 4 5` 
3. save and exit
4. reopen world

This causes an array out of bounds error when joining the world.
I believe this is because there is no protection when loading an invalid constelation. the constellation in the saved constellation file is
```
2 3 4 5 | 
```
there should probably be a check to make sure a line is valid before attempting to load the constellation

**Comment by Nettakrim** [comment]
yeah that's why it's part of the scary sounding /sga:admin command :P

i should easily be able to fix that, i wonder if it also happens with editing a /sga:share message?

late response as somehow i didn't see this issue until 5 minutes ago

**Comment by Nettakrim** [comment]
fixed, it will be in the next update for 1.19.3+
